addons/website_quote/models/sale_order.py

```diff
@@ -51,6 +51,13 @@ def _get_default_template(self):
         template = self.env.ref('website_quote.website_quote_template_default', raise_if_not_found=False)
         return template and template.active and template or False
 
+    def _get_default_online_payment(self):
+        if self.env['ir.config_parameter'].sudo().get_param('sale.sale_portal_confirmation_options', default='none') == 'pay':
```

Comment:
It could be simplified by
```
return 1 if self.env['ir.config_parameter'].sudo().get_param('sale.sale_portal_confirmation_options', default='none') == 'pay' else 0
```

Reply:
This was the previous version, but as I created a new function for it, I want to do it on separate lines to more clarity... I can simplify it again if needed